rust/template/cmd_parser/Cargo.toml

```diff
@@ -1,15 +1,16 @@
 [package]
 name = "cmd_parser"
 version = "0.1.0"
+edition = "2018"
 
 [dependencies.differential_datalog]
 path = "../differential_datalog"
 
 [dependencies]
+libc = "0.2.42"
```

Comment:
Do we need to specify the minor version of each package given that after 1.0 they should maintain compatibility across minor versions?

Reply:
I don't think so. I was not sure about this case, though. Typically that is done if a new feature was introduced and we want to make sure that a version with this feature is consumed. I don't have the background to judge whether that was the case for `libc` here.
As for the others, I don't think it makes a difference here ([reference](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements)). Do you want me to shorten the numbers where possible?